src/dashboard.tsx

```diff
@@ -301,9 +301,11 @@ export class URLInput extends Widget {
   /**
    * The in browser dashboard check for authenticated dashboards.
    */
+  get browserDashboardCheck(): boolean {
+    return this.browserDashboardCheck;
```

Comment:
These should be underscored attributes to not shadow the method names

Reply:
Sorry to be terse! Typically what we do is add a private attribute (see the bottom of this class) which is prefixed with an underscore. It's a bit Java-ey, but hopefully not too bad.

So this line would become
```typescript
return this._browserDashboardCheck;
```
and further down with the other private attributes you would have

```typescript
private _browserDashboardCheck = false;
```

Reply:
Cool, just added that. Let me know if its correct :slightly_smiling_face: